src/netbase.cpp

```diff
@@ -651,7 +667,7 @@ void CNetAddr::SetRaw(Network network, const uint8_t *ip_in)
             memcpy(ip, ip_in, 16);
             break;
         default:
-            assert(!"invalid network");
+	    assert(true);  // invalid network
```

Comment:
Let's have an explicit function to abort with a message, rather than misusing assert for this.